assets/js/components/data.js

```diff
@@ -251,7 +252,7 @@ const dataAPI = {
 			this.maxRequests = 10;
 		}
 
-		const { datacache } = googlesitekit.admin;
+		const datacache = null !== getQueryParameter( 'datacache' );
 		return wp.apiFetch( {
 			path: addQueryArgs( `/google-site-kit/v1/data/${ datacache ? '?datacache' : '' }`,
 				{
```

Comment:
Shouldn't `datacache` be part of the object of query args rather than part of the URL's path in `addQueryArgs`?

Reply:
`datacache` is a special thing - it only exists for our internal purposes of regenerating the cache data for visual regression tests. It shouldn't be included in the actual data, as it shouldn't influence the cache key. Even better would be to remove it and implement some mini utility plugin to generate the cache data - but that is something for another issue.